consurg/guard/tui.py

```
@@ -104,21 +104,29 @@ def _build_approval(state: GuardState) -> Panel | None:
     return Panel(text, title="\u26a1 APPROVAL REQUIRED", border_style="yellow")
 
 
+def _build_footer(state: GuardState) -> Panel:
```

Ruff (0.15.0)

[warning] Unused function argument: `state`

(ARG001)